api/api.go

```diff
@@ -292,3 +303,30 @@ func (c *Client) Skip(language, slug string) error {
 
 	return errors.New(pe.Error)
 }
+
+// Status sends a request to exercism to fetch the user's
+// completion status for the given language track.
+func (c *Client) Status(language string) (*StatusInfo, error) {
+	c.XAPIHost = "http://localhost:9393"
```

Comment:
For reference, you can edit the config file to edit the xapi host (by default, that's `~/.exercism.json`, unless you have one of the environment variables defined).